code/controllers/subsystem/ticker.dm

```diff
@@ -571,15 +571,18 @@ SUBSYSTEM_DEF(ticker)
 	update_everything_flag_in_db()
 	if(!round_end_sound)
 		round_end_sound = pick(\
-		'sound/roundend/newroundsexy.ogg',
-		'sound/roundend/apcdestroyed.ogg',
-		'sound/roundend/bangindonk.ogg',
-		'sound/roundend/leavingtg.ogg',
-		'sound/roundend/its_only_game.ogg',
-		'sound/roundend/yeehaw.ogg',
-		'sound/roundend/disappointed.ogg',
-		'sound/roundend/scrunglartiy.ogg',
-		'sound/roundend/petersondisappointed.ogg'\
+		'sound/roundend/deliguana.ogg',
+		'sound/roundend/undecided.ogg',
+		'sound/roundend/repair.ogg',
+		'sound/roundend/knockknock.ogg',
+		'sound/roundend/flow.ogg',
+		'sound/roundend/gorp.ogg',
+		'sound/roundend/nuts.ogg',
+		'sound/roundend/boowomp.ogg',
+		'sound/roundend/helpme.ogg',
+		'sound/roundend/shiptestingthursday.ogg',
+		'sound/roundend/shitpest.ogg',
+		'sound/roundend/gayrights.ogg'\
```

Comment:
Took a listen, not too interested in the "meme-y" ones, good ones though are:

`repair.ogg`.
`deliguana.ogg.`
`gayrights.ogg`
`shiptestingthursday.ogg`
`boowomp.ogg`
`undecided.ogg`